### DISCUSSION TITLE
Run in sequence

### DISCUSSION THREAD

**Discussion by Kri72** [home-assistant/frontend#18269] [Automations / scripts / scenes]
Hello, The current “Run in parallel” action in the automation editor, only enable to run single actions in parallel, when sometimes one might want to run in parallel several sequences.
I would propose to add an “Run in sequence” action, that can possibly be used in the “Run in parallel” action to gather several actions to be run sequentially, in parallel of one of several others…
I know there are ways around in creating scripts to gather these sequential actions, but i think it is easier to have all in the automation rather than cutting it in slices...

**Comment by karwosts** [comment]
Being worked on https://github.com/home-assistant/frontend/pull/18470
